docs/products/postgresql/howto/check-iop.rst

```diff
@@ -0,0 +1,57 @@
+IOP capability of plans
+-----------------------
+
+This article will explain what is the typical IOP capability used for common plans in pg:
+
+Hobbyist
+''''''''
+
+==============================     ================     ===================     ================
+  Plan                                AWS                  GCP                     Azure
+==============================     ================     ===================     ================
+ Hobbyist                            t3.micro             g1-small                 not available                
+==============================     ================     ===================     ================
+
+
+Startup
+'''''''
+
+==============================     ================     ===================     ================
+  Plan                                AWS                  GCP                     Azure
+==============================     ================     ===================     ================
+ Startup-4                          t3.medium            e2-medium               standard_d1_v2                
+ Startup-8                          m5.large             e2-standard-2           standard_d2_v2
```

Comment:
some places is written like `Startup` and some places like `startup`. I think is better to keep consistency so we can pick one. But considering that the rest of the words are with lowercase, then maybe we go with lowercase, what do you think?

Reply:
changed to startup